src/freakble/ble.py

```diff
@@ -82,7 +88,8 @@ def on_disconnect(self, client: BleakClient):
 
     async def send(self, data):
         if self._client is not None:
-            await self._client.write_gatt_char(NORDIC_UART_TX_CHARACTERISTIC, data)
+            ans = await self._client.write_gatt_char(self.uart_tx_char, data, response=True)
+            logging.debug(self.uart_tx_char.uuid, ans)
```

Comment:
We discovered that response=True don't change, as you discovered it seems that on MacOS just the uuid of characteristics are inverted. I don't know why.